app/controllers/api/timeslots_controller.rb

```diff
@@ -15,9 +15,11 @@ def create
   end
 
   def update
-    @timeslot = Timeslot.find_by(id: safe_params[:id])
-    @timeslot.student_id = current_user.id
-    if @timeslot.save
+    timeslot = Timeslot.find_by(id: safe_params[:id])
+    timeslot.student_id = current_user.id
```

Comment:
Do you think I should add that kind of check now or add it as an issue to be addressed?